Fix admin-api-namespaces page grammatical errors (#7779)
apache/pulsar: site2/docs/admin-api-namespaces.md

#### set backlog quota policies

> Backlog quota helps the broker to restrict bandwidth/storage of a namespace once it reach a certain threshold limit. Admin can set this limit and one of the following actions after the limit is reached.

Reviewer: ```suggestion
Backlog quota helps the broker to restrict bandwidth/storage of a namespace once it reaches a certain threshold limit. Admin can set the limit and take corresponding action after the limit is reached.
```

  1.  producer_request_hold: broker will hold and not persist produce request payload

>   2.  producer_exception: broker will disconnect with the client by giving an exception

Reviewer: could you please refine all of them as present tense (rather than future tense)? Thank you.
Usually, we use present tense in technical writing. For details, refer to https://developers.google.com/style/tense

>   2.  producer_exception: broker will disconnect with the client by giving an exception

Reviewer 2: ```suggestion
  2.  producer_exception: broker disconnects with the client by giving an exception.
```

#### unload namespace bundle

> Namespace bundle is a virtual group of topics which belong to the same namespace. If broker gets overloaded with the number of bundles then this command can help to unload heavy bundle from that broker, so it can be served by some other less loaded broker. Namespace bundle is defined with it’s start and end range such as 0x00000000 and 0xffffffff.

Reviewer 2: ```suggestion
The namespace bundle is a virtual group of topics which belong to the same namespace. If the broker gets overloaded with the number of bundles, this command can help unload heavy bundle from that broker, so it can be served by some other less-loaded brokers. The namespace bundle ID ranges from 0x00000000 to 0xffffffff.
```

#### set retention

> Each namespace contains multiple topics and each topic’s retention size (storage size) should not exceed a specific threshold or it should be stored till a certain time duration. This command helps to configure retention size and time of topics in a given namespace.

Reviewer 2: ```suggestion
Each namespace contains multiple topics and the retention size (storage size) of each topic should not exceed a specific threshold or it should be stored for a certain period. This command helps configure the retention size and time of topics in a given namespace.
```

#### set dispatch throttling

It sets message dispatch rate for all the topics under a given namespace. 

> Dispatch rate can be restricted by the number of messages per X seconds (`msg-dispatch-rate`) or by the number of message-bytes per X second (`byte-dispatch-rate`).

Reviewer 2: ```suggestion
The dispatch rate can be restricted by the number of messages per X seconds (`msg-dispatch-rate`) or by the number of bytes per X second (`byte-dispatch-rate`).
```

#### set dispatch throttling for subscription

It sets message dispatch rate for all the subscription of topics under a given namespace.

> Dispatch rate can be restricted by the number of messages per X seconds (`msg-dispatch-rate`) or by the number of message-bytes per X second (`byte-dispatch-rate`).

Reviewer 2: ```suggestion
The dispatch rate can be restricted by the number of messages per X seconds (`msg-dispatch-rate`) or by the number of bytes per X second (`byte-dispatch-rate`).
```

#### set dispatch throttling for replicator

It sets message dispatch rate for all the replicator between replication clusters under a given namespace.

> Dispatch rate can be restricted by the number of messages per X seconds (`msg-dispatch-rate`) or by the number of message-bytes per X second (`byte-dispatch-rate`).

Reviewer 2: ```suggestion
The dispatch rate can be restricted by the number of messages per X seconds (`msg-dispatch-rate`) or by the number of bytes per X second (`byte-dispatch-rate`).
```

> Dispatch rate can be restricted by the number of messages per X seconds (`msg-dispatch-rate`) or by the number of message-bytes per X second (`byte-dispatch-rate`).

Reviewer 2: ```suggestion
The dispatch rate can be restricted by the number of messages per X seconds (`msg-dispatch-rate`) or by the number of bytes per X second (`byte-dispatch-rate`).
```
